### ISSUE TITLE
Server in ERRORED state when loading single player

### ISSUE THREAD

**Issue by tyra314** [ThunderModder/bionisation3#1]
Hi,

your mod looks interesting and while trying it out, I got a problem.
I'm using a custom pack with:
- Minecraft 1.12.2
- Forge: 14.23.2.2618
- Bionisation 3: 1.12.2-1.0.0
- EvilCraft: 1.12.2-0.10.47

The game loads just fine into the main menu. When I go into Single Player and create a new world, the server goes into the ERRORED state, but the client window keeps running. Therefore, there is no crash report to link. However, here is the log file: https://paste.ee/p/wnk5k

I also tested it with a new instance with only Bionisation 3 and EvilCraft installed as mods and it also happens. So there's probably something going on there between those two.

**Comment by ThunderModder** [comment]
Hi! Thank you for the report! It seems the problem is in EvilCraft:
`Caused by: java.lang.NullPointerException
	at org.cyclops.evilcraft.item.WerewolfFlesh.func_77624_a(WerewolfFlesh.java:175) ~[WerewolfFlesh.class:?]`
Can you try without this mod?

**Comment by tyra314** [comment]
I'd rather say the problem gets visible with EvilCraft. I tried it in another pack without EvilCraft. This time, quark struggles, but it's still pretty similar if you'd ask me:

https://gist.github.com/tyra314/4d8779368dd6359da0d96be1b99694f3

```
Caused by: java.lang.NullPointerException
	at vazkii.quark.misc.item.ItemSlimeBucket.func_77653_i(ItemSlimeBucket.java:104) ~[ItemSlimeBucket.class:?]
	at net.minecraft.item.ItemStack.func_82833_r(ItemStack.java:617) ~[aip.class:?]
	at net.minecraft.item.ItemStack.func_82840_a(ItemStack.java:662) ~[aip.class:?]
	at net.minecraft.client.Minecraft.lambda$populateSearchTreeManager$1(Minecraft.java:575) ~[bib.class:?]
```

**Comment by ThunderModder** [comment]
Hmm, this is an issue. I will try to locate and fix it

**Comment by ThunderModder** [comment]
Fixed in version 1.0.1. Thank you for bug report.
